_data/companies.yml

```diff
@@ -1434,11 +1434,11 @@
   last_update: 2020-03-04
 
 - name: Slido
-  wfh: Encouraged
+  wfh: Restricted
```

Comment:
Could you explain what restricted means in this sense?

Reply:
Jeez, nice catch, that's a typo. Thx!

```suggestion
  wfh: Required
```